Marlin/src/gcode/motion/M290.cpp

```diff
@@ -87,6 +91,28 @@ void GcodeSuite::M290() {
       #endif
     }
   #endif
+
+  if (!parser.seen("XYZ")) {
+    SERIAL_ECHO_START();
+    #if ENABLED(BABYSTEP_HOTEND_Z_OFFSET)
+      SERIAL_ECHOLNPAIR(MSG_PROBE_OFFSET MSG_Z ": ", probe_offset[Z_AXIS]);
+      #if ENABLED(BABYSTEP_XY)
+        SERIAL_ECHOLNPAIR(MSG_X_OFFSET ": ", hotend_offset[X_AXIS][active_extruder]);
+        SERIAL_ECHOLNPAIR(MSG_Y_OFFSET ": ", hotend_offset[Y_AXIS][active_extruder]);
+      #endif
+      SERIAL_ECHOLNPAIR(MSG_Z_OFFSET ": ", hotend_offset[Z_AXIS][active_extruder]);
```

Comment:
We can't use `MSG_Z_OFFSET` in serial output. It is a translatable. I may rename all translatable strings with an "`LCD_`" prefix to make sure these typos are less common.